common/k8stest/util_rdma.go

```diff
@@ -317,3 +317,41 @@ func GetDevLinkName(node, iface string) (string, error) {
 	}
 	return "", nil
 }
+
+func DisableRdmaDeicesOnAllMayastorNodes() error {
+	workerNodes, err := ListIOEngineNodes()
```

Comment:
IMO should return here immediately if `e2e_config.GetConfig().DisabledRdmaDevices` is empty

Reply:
done[https://github.com/openebs/openebs-e2e/compare/97f6bb9f1285bb8f08fe736bb498948de52c0b3f..7a4e8d1886a4b1bebeba350d30dde9c35e2ec096](url)